collections/_learn/overview.md

```diff
@@ -0,0 +1,104 @@
+
+Effects are typed payloads that essentially act as an **instruction set** for the EMR on how to persist data or modify the user's experience.
+
+**Effect Examples and Corresponding Events:**
+
+| **Triggering Event** | **Corresponding Effect(s)** |
+| --- | --- |
+| **A patient was created** | Create a task for staff to send a welcome packet. |
+| **User views a select box** | Present filtered, reordered, and annotated options instead of the default. |
+| **Dr. Smith started a prescription** | Pre-fill the prescription details based on a structured form or patient data. |
+| **A custom application was launched** | Launch a specific modal or pre-fill chart notes based on a template. |
+| **It is 7:05 pm (Cron)** | Rebalance task assignment for overdue tasks. |
+| **Received an HTTP POST to `<path>`** | Return a custom JSON response with a specific payload and status code. |
+
+The platform aims to give plugins the same capabilities as end-users, with the key exception being regulatory limitations. For example, a plugin can automate the filling out of a prescription, but a prescriber must still manually sign off on it—the final click of approval must come from a human.
+
+### Implementing Your Logic: Types of Handlers
+
+Your plugin is built around **Handlers**, which are the specific units of code you provide. They may listen for events by declaring them directly or by inheriting from a specific handler subclass.
+
+Key Handler Types to Know:
+
+- **`BaseHandler`**: The most fundamental type. You declare the events it listens for in the `RESPONDS_TO` constant and implement the `compute` method to return a list of effects.
+- **`CronTask`**: Designed for scheduled automation. You set the cron string in the `SCHEDULE` constant and implement the `execute` method.
+- **`ActionButton`**: Allows you to customize the EMR user interface. You set the button label and location and implement the `handle` method. An optional `visible` method can be implemented to make the button appear only under certain conditions.
+- **`SimpleAPI`**: For custom HTTP interactions. You implement an `authenticate` method and a method for each request path, annotated with the path and verb (e.g., GET, POST), resulting in a custom endpoint IN YOUR CANVAS INSTANCE…!
+- **`Application`**: Used to embed custom user interfaces or launch modals within the EMR. You provide an icon and title, and implement the `on_open` method.
+
+These handlers allow you to build everything from small quality-of-life automations (like [AI note titles](https://docs.canvasmedical.com/sdk/example-ai_note_titles/)) to complex integrations ([Syncing patients between systems](https://docs.canvasmedical.com/sdk/example-patient_creation_platform_sync/)).
+
+### Summary
+
+In this overview, you learned about:
+
+- The two primary ways to extend the Canvas EMR: the **FHIR API** for general interoperability and the **Canvas SDK** for deep system customization via Python plugins.
```

Comment:
one thing that wasn't really covered in this section is that a single plugin can contain many different handlers/protocols. maybe something about how a single plugin can group handlers of similar functionality together.